lib/rules/jsx-no-comment-textnodes.js

```diff
@@ -36,19 +36,15 @@ function checkText(node, context) {
 module.exports = {
   meta: {
     docs: {
-      description: 'Comments inside children section of tag should be placed inside braces',
+      description: 'Disallow comments from being inserted as text nodes',
       category: 'Possible Errors',
       recommended: true,
       url: docsUrl('jsx-no-comment-textnodes'),
     },
 
     messages,
 
-    schema: [{
-      type: 'object',
-      properties: {},
-      additionalProperties: false,
-    }],
+    schema: [],
```

Comment:
i think this is actually undesirable - does the latter actively prevent users from configuring anything?

Reply:
`schema: []` validates that no options are passed to the rule. If any options (garbage) are passed, ESLint will throw an error.

More info: https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/main/docs/rules/require-meta-schema.md

Reply:
aha, ok thanks. that's great then

Reply:
is that true on older versions of eslint as well?

Reply:
Yes, this has been the case since ESLint v2:
* https://eslint.org/blog/2016/02/eslint-v2.0.0-released/
* https://github.com/eslint/eslint/pull/4790